persistence-modules/read-only-transactions/src/main/java/com/baeldung/read_only_transactions/h2/Config.java

```diff
@@ -0,0 +1,52 @@
+package com.baeldung.read_only_transactions.h2;
```

Comment:
Although the underscore is compliant with the package naming convention we discourage its use, so could you rename the package?